packages/fe-fpm-writer/test/unit/column.test.ts

```diff
@@ -79,7 +79,47 @@ describe('CustomAction', () => {
                 width: '150px',
                 properties: ['ID', 'TotalNetAmount', '_CustomerPaymentTerms/CustomerPaymentTerms']
             };
-            generateCustomColumn(testDir, { ...testCustomColumn, minUI5Version: '1.86' }, fs);
+            generateCustomColumn(testDir, { ...testCustomColumn, minUI5Version: '1.86.3' }, fs);
+            const updatedManifest = fs.readJSON(join(testDir, 'webapp/manifest.json')) as Manifest;
+            const settings = (
+                updatedManifest['sap.ui5']?.['routing']?.['targets']?.['sample']?.['options'] as Record<string, any>
+            )['settings'];
+            expect(settings.controlConfiguration).toMatchSnapshot();
+
+            expect(fs.read(expectedFragmentPath)).toMatchSnapshot();
+            expect(fs.read(expectedFragmentPath.replace('.fragment.xml', '.js'))).toMatchSnapshot();
+        });
+
+        test('version 2.0.0, with new handler, all properties', () => {
```

Comment:
What is the reason for this test case? It looks like copy and paste from the last one just with a different version? Could we slim this one down to actually only verify the version relevant aspects? or consider a `test.each` if we need to run the very same test with multiple versions.

Reply:
Yes, I can introduce test.each here.  2.0.0 as I wanted to have at least one test case for the scenario described by Klaus before, i.e. a potential major version bump of UI5.

Reply:
Hi @schreckstefan , you could also create unique titles by using parameters https://jestjs.io/docs/api#testeachtablename-fn-timeout
See example here: https://dev.to/bgord/simplify-repetitive-jest-test-cases-with-test-each-310m

Reply:
Thus the question is if I should not replace test.each...

Reply:
Thanks @hitesh-parmar , I applied this to all test.each now where I could see identical texts.

Reply:
ok, not to all: weird, but in some cases the use of %p or &p led to falsification of the parameter data, and to wrong snapshot results. Made undone, and the snapshots look fine. In other tests this falsification does not occur. I have no explanation for this, but only used the %p or &p syntax now where it did not lead to falsifications.